src/App.tsx

```diff
@@ -74,9 +75,11 @@ export function App() {
 
   return (
     <AppProvider theme={theme} i18n={enTranslations}>
-      <Frame topBar={topBarMarkup} navigation={navigationMarkup}>
-        {getCurrentMarkup()}
-      </Frame>
+      <RestfulProvider base={process.env.REACT_APP_API_URL ?? ""}>
```

Comment:
Assigning an empty string will cause an unexpected and somewhat non-obvious error when we attempt to fetch, will it not? If we are missing the env var, it would probably be better to throw an explicit error detailing the cause, show a helpful message, and/or log it.

Reply:
I was thinking this would default to the production API once available? But if that doesn't make sense I agree that it should throw an explicit error.